pkg/pod/resource_request.go

```diff
@@ -50,16 +51,45 @@ func resolveResourceRequests(containers []corev1.Container) []corev1.Container {
 		}
 	}
 
+	// Get limitrange minimum for container requests so they won't
+	// be zeroed out if minimum is specified in namespace
+	var limitRangeItems []corev1.LimitRangeItem
+	if limitRange != nil {
+		limitRangeItems = limitRange.Spec.Limits
+	}
+	min := allZeroQty()
+	if limitRange != nil {
+		for _, limitRangeItem := range limitRangeItems {
+			if limitRangeItem.Type == "Container" {
```

Comment:
minor: suggest `== corev1.LimitTypeContainer` rather than the literal string "Container" here.